File: device-settings/src/org/lineageos/device/settings/refreshrate/RefreshRateActivity.java
/*
 * SPDX-FileCopyrightText: 2025 AlphaDroid
 * SPDX-License-Identifier: Apache-2.0
 */

package org.lineageos.device.settings.refreshrate;

import android.os.Bundle;

import com.android.settingslib.collapsingtoolbar.CollapsingToolbarBaseActivity;

import org.lineageos.device.settings.R;

public class RefreshRateActivity extends CollapsingToolbarBaseActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_refresh_rate);
        setTitle(getString(R.string.refresh_rate_title));
    }
}

File: device-settings/src/org/lineageos/device/settings/refreshrate/RefreshRateActivity.kt
/*
 * SPDX-FileCopyrightText: 2025 AlphaDroid
 * SPDX-License-Identifier: Apache-2.0
 */
package org.lineageos.device.settings.refreshrate

import android.os.Bundle
import androidx.activity.ComponentActivity
import org.lineageos.device.settings.ui.DeviceSettingsDest
import org.lineageos.device.settings.ui.setDeviceSettingsContent

class RefreshRateActivity : ComponentActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setDeviceSettingsContent(DeviceSettingsDest.Refresh)
    }
}
